blocks/form/form.js

```diff
@@ -526,6 +526,9 @@ export default async function decorate(block) {
       if (block.classList.contains('gdpr')) {
```

Comment:
To avoid confusion in the future we should rename this to `gdpr-link-generation`